zng/docs/spec.md

```diff
@@ -180,8 +180,7 @@ These payloads are guaranteed to be preserved
 in order within the stream and presented to higher layer components through
 any ZNG streaming API.  In this way, senders and receivers of ZNG can embed
 protocol directives as ZNG control payloads rather than defining additional
-encapsulating protocols.  See the
-[zng-over-http](zng-over-http.md) protocol for an example.
+encapsulating protocols.
```

Comment:
The "ZNG over HTTP" doc as it stands today is yet another artifact from a "ZSON" time when things were all text-based and we were attempting genuine backward-compatibility with Zeek TSV. While it's surely helpful to point to a reference implementation for anything described in the spec, since the [zeek-tsv-http-plugin](https://github.com/brimsec/zeek-tsv-http-plugin) is not ZNG in the modern sense of the word, it seems like we should probably do away with it. Opinions?

Reply:
I agree we should do away with it.

Reply:
Thanks for the confirmation @henridf. I've now removed the doc itself and another link to it in a follow-on commit.